app-admin/dio/dio-1.5.2-r1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit toolchain-funcs
+
+DESCRIPTION="A device I/O monitoring tool"
+HOMEPAGE="https://github.com/donaldmcintosh/dio"
+SRC_URI="https://github.com/donaldmcintosh/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="MIT"
+SLOT="0"
+
+RDEPEND="sys-libs/ncurses:0="
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/${P}/src"
+
+DOCS=( "README" "../README.md" )
+HTML_DOCS=( "../site/dio.png" "../site/www.diodio.org" )
```

Comment:
What's the purpose of `dio.png`?

Reply:
I thought, it belongs to the html docs, but apparently not, so dropped.